src/main/java/net/coderbot/iris/mixin/MixinLevelRenderer.java

```diff
@@ -187,6 +187,15 @@
 		pipeline.pushProgram(GbufferProgram.WEATHER);
 	}
 
+	@Inject(method = RENDER_WEATHER, at = @At(value = "INVOKE", target = "Lcom/mojang/blaze3d/systems/RenderSystem;depthMask(Z)V", shift = At.Shift.AFTER))
+	private void iris$writeRainAndSnowToDepthBuffer(LightTexture lightTexture, float tickDelta, double d, double e, double g, CallbackInfo callback) {
```

Comment:
This injection functions properly, however it might be nice to make sure that it only applies to the first depthMask call in this function. While renderWorldBounds sets up its expected depthMask state anyways, it's nice to be as precise as possible with our injections. For targeting the first injection, an ordinal=0 is acceptable.

Reply:
Can we not modify variable?

Reply:
You might be thinking of ModifyArg. I suppose that's possible too.